content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
+   - Integrations with external software (EDI, APIs, etc.)
+   - Workflows between different apps (online sales with eCommerce, converting a lead all the way to
+     a sales order, delivery of products, etc.)
+   - Data exports
+   - Automated actions
+   - Server actions in the action menu on form views, as well as by selecting multiple records on
+     list views
 
-.. _upgrade/assistance:
+Upgrading the production database
+---------------------------------
 
-Help
-====
+Once the :ref:`tests <upgrade/test_your_db>` are completed and you are confident that the upgraded
+database can be used as your main database without any issue, it is time to plan the go-live day. It
+can be planned in coordination with Odoo's upgrade support analysts, reachable via the `support page
+<https://odoo.com/help>`_.
 
-.. _upgrade/contact:
+.. important::
+   - Any modification to your production database will **not** be saved during the upgrade of your
+     production database. This is why we recommend not using it during that time.
+   - Going into production without first testing may lead to:
 
-Contact our upgrade service support
------------------------------------
+     - Users failing to adjust to the changes and new features
+     - Business interruptions (e.g., no longer having the possibility to validate an action)
+     - Poor customer experience (e.g., an eCommerce website that does not work correctly)
 
-Should you have any more questions about the upgrade, do not hesitate to send a message to `Odoo
-Upgrade Team <mailto:upgrade@odoo.com>`_. We will be happy to answer it as soon as possible.
+The process of upgrading a production database is similar to upgrading a test database with a few
+exceptions.
 
-.. _upgrade/supported-versions:
+.. tabs::
 
-Supported versions
-------------------
+   .. group-tab:: Odoo Online
 
-Please note that Odoo provides support and bug fixing only for the three last major versions of
-Odoo.
+      The process is equivalent to :ref:`obtaining an upgraded test database
+      <upgrade/request-test-database>`, except for the purpose option, which must be set to
+      :guilabel:`Production` instead of :guilabel:`Test`.
 
-This is a factor to take into consideration before upgrading. If you are on an older version, we
-suggest you to prefer the most recent version to benefit from longer support (before having to
-upgrade again).
+      .. warning::
+         Once the upgrade is requested, the database will be unavailable until the upgrade is
+         finished. Once the process is completed, it is impossible to revert to the previous
+         version.
 
-.. seealso::
-   :doc:`maintain/supported_versions`
+   .. group-tab:: Odoo.sh
 
-.. _upgrade/sla:
+      The process is equivalent to :ref:`obtaining an upgraded test database
+      <upgrade/request-test-database>`, except that the :guilabel:`Production` branch must be
+      selected before clicking the :guilabel:`Upgrade` tab.
```

Comment:
`except that the :guilabel:`Production` branch must be selected before clicking the :guilabel:`Upgrade` tab.`
-> 
`on the :guilabel:`Production` branch` 

What do you think ?

Reply:
Maybe I'd say: `The process is similar to...` instead of `equivalent`
For me (and I'd rather avoid confusion for other people reading the doc as well), "equivalent" means that it's doing the same.
While in fact, even if the button is the same, it's not triggered by the same factors (as it's explained after).
That's why I would prefer "similar", so it underlines that it's not the same, it might resemble, but it's not the same.

Also in the change you put ```on the :guilabel:`Upgrade` branch.``` :see_no_evil: